ote/src/clj/ote/authorization.clj

```diff
@@ -25,11 +25,17 @@
                          {:status 403 :body "Forbidden"})]
     (cond
       (nil? transport-operator-id)
-        (access-denied)
+        (do
+          (log/debug "ACCESS Denied for user " (pr-str user) " operator-id " transport-operator-id)
+          (access-denied)
+        )
 
       (and (not is-admin?)
            (not (contains? allowed-operators transport-operator-id)))
+      (do
+        (log/debug "ACCESS Denied for user " (pr-str user) " operator-id " transport-operator-id)
```

Comment:
The same logging call is in both places that call `(access-denied)` move it there. 
Closing parens.